concourse/pipelines/linux-image-build.jsonnet

```diff
@@ -476,9 +476,9 @@ local imggroup = {
              [common.gcsimgresource { image: image, gcs_dir: 'rocky-linux' } for image in rocky_linux_images] +
              [common.gcssbomresource { image: image, sbom_destination: 'rocky-linux' } for image in rocky_linux_images] +
              [common.gcsshasumresource { image: image, shasum_destination: 'rocky-linux' } for image in rocky_linux_images] +
-             [common.gcsimgresource { image: image, gcs_dir: 'accelerators' } for image in accelerator_images] +
-             [common.gcssbomresource { image: image, sbom_destination: 'accelerators' } for image in accelerator_images] +
-             [common.gcsshasumresource { image: image, shasum_destination: 'accelerators' } for image in accelerator_images] +
+             [common.gcsimgresource { image: image, gcs_dir: 'accelerators' } for image in rocky_accelerator_images] +
```

Comment:
Lets put these SBOMs with the rest of the rocky images, actually. I want to make this just another rocky image as much as possible.

Reply:
Sure, done